test/e2e/allocator_test.go

```diff
@@ -119,11 +128,26 @@ func TestAllocatorCrossNamespace(t *testing.T) {
 
 	// Create namespaces A and B
 	namespaceA := fmt.Sprintf("allocator-a-%s", uuid.NewUUID())
-	framework.CreateNamespace(t, namespaceA)
-	defer framework.DeleteNamespace(t, namespaceA)
+	err := framework.CreateNamespace(namespaceA)
```

Comment:
namespaceA can be framework.Namespace. 
namespaceA := framework.Namespace
No need to create a new one.

Reply:
done